libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/Notification.java

```diff
@@ -26,12 +26,12 @@
     // key value pair describing additional details about notification
     private Map<String, String> details;
 
-    // Type of notification
-    private String type;
-
     // Utility class to convert the notification into an email
     private NotificationToEmailConverter notificationToEmailConverter;
 
+    // Utility class to convert the notification into metric attributes
+    private NotificationToMetricConverter notificationToMetricConverter;
```

Comment:
NotificationToMetricConverter purpose is similar to NotificationToEmailConverter (instead of mail convert to metrics)